README.md

```diff
@@ -25,15 +25,21 @@ After cloning this project, run the following command to install dependencies:
 yarn # or 'npm install'
 ```
 
-Next, you'll need a MySQL server running and a connection string available as environment variable `DATABASE_URL`.
-You can use `docker-compose` to set up default MySQL server in container:
+Next, you'll need a Postgres server running and a connection string available as environment variable `DATABASE_URL`.
+You can use `docker-compose` to set up default Postgres server in container:
 
 ```bash
 docker-compose up -d
 ```
 
 > For local development, you can create a .env file from the .env.example file and the application will read the values on startup.
 
+Then, build:
+
+```bash
+yarn build
+```
+
```

Comment:
```suggestion
```

We don't need to build in local development, `yarn dev` handles everything.

Reply:
if just running `yarn dev` after cloning and running `yarn` errors like so:

<img width="799" alt="Screenshot 2025-02-26 at 15 16 36" src="https://github.com/user-attachments/assets/14a9536b-03fb-43d7-8601-c8560875ca50" />

that's why I added `yarn build`. Maybe I'm doing something wrong?

Reply:
Oh I see. We have `codegen` step that generates ORM models based on your schema. And you need to run it before you can run the code.

I think we can just add some note saying that every time you modify your schema you should run `yarn codegen`.

Reply:
Is there a way to make it simpler? Maybe have `yarn dev` run `yarn build` or `yarn codegen`? 

If not, should we simply keep the "yarn build" step in the README?

Reply:
Let's add `predev` script that runs `codegen` (similar to prebuild).

Reply:
a7837ecc44e40fbb646e4f2e8762230b32ea3cc9